src/domain/users/entities/user.entity.ts

```diff
@@ -2,7 +2,6 @@ import { z } from 'zod';
 import { RowSchema } from '@/datasources/db/v1/entities/row.entity';
 
 export enum UserStatus {
-  PENDING = 0,
```

Comment:
We need this PENDING status for when a wallet is invited to an org ([Spec](https://www.notion.so/safe-global/Data-flows-1858180fe5738004a4adc09d03ae4b7a?pvs=4#1858180fe57380528f35ceb4bf97e2db))